src/components/Tile/Tile.tsx

```diff
@@ -4,41 +4,26 @@ import { Paragraph } from "../typography/Paragraph";
 import "./Tile.scss";
 export interface TileProps {
   /**
-   * Array of {@link TileTarget} objects that defines the number of tiles in the app
-   */
-  targets: TileTarget[];
-}
-interface TileTarget {
-  /**
-   * An object with parameters for an address to hyperlink to, a title to name that link and optional children props to pass into the component
+   * Props to be passed in to Tile component and rendered.
    */
   link?: string;
   title?: string;
   children?: ReactNode;
 }
+
 export const Tile = (props: TileProps): React.ReactElement => {
-  const renderTiles = (targets: TileTarget[]): React.ReactElement[] => {
-    return targets.map((tile) => (
-      <div className="govuk-grid-row" key={tile.link}>
-        <div className="govuk-grid-column-one-quarter">
-          <div className="tile" data-test="tile-container">
-            <Heading level={HeadingLevels.H3}>
-              <a href={tile.link} data-test="tile-link">
-                {tile.title}
-              </a>
-            </Heading>
-            <Paragraph>{tile.children}</Paragraph>
-          </div>
+  return (
+    <div className="govuk-grid-row" key={props.link}>
+      <div className="govuk-grid-column-one-quarter">
```

Comment:
Not sure that we want any grid work in the component.
Feels like it should be up to whoever's using the component to decide on the grid system and how to arrange it?